website/blog/2023-11-09-IPv6-addresses/item.md

```diff
@@ -0,0 +1,223 @@
+
+We use domain names for websites, to avoid having to remember IP addresses, though our readers who are sysadmin—used to referencing all sorts of nodes deep within their organization’s networks—will recall them at the drop of a hat.
+
+But, increasingly, since many devices are online and [96.6% of internet users now use a smartphone](https://www.oberlo.com/statistics/how-many-people-have-smartphones), most Internet of Things (IoT) devices that we have in our workplaces and homes _also_ have their own IP address. This includes:
+
+-   Computers, laptops and smartphones
+-   Database servers, web servers, mail servers, virtual servers (virtual machines), and servers that store software packages for distribution
+-   Other devices such as network printers, routers and services running on computer networks
+-   Domain names for websites, which are mapped to the IP address using Domain Name Servers (DNS)
+
+IP addresses are centrally overseen by the Internet Assigned Numbers Authority ([IANA](https://www.iana.org/)), with five [Regional Internet Registries](https://www.nro.net/about/rirs/) (RIRs).
+
+## What is the state of the IP landscape right now?
+
+Well, it’s all down to numbers.
+
+The previous version of this network layer communications protocol is known as IPv4. From our informed vantage point—looking over the rapid growth of ecommerce, business, government, educational, and entertainment services across the internet—it’s easy to see how its originator could not possibly have predicted that demand for IPv4 addresses would outstrip supply.
+
+Add in the ubiquity of connected devices that allow us to access and consume those services and you can see the problem.
+
+IP address exhaustion was foreseen in the 1980s, which is why the Internet Engineering Task Force ([IETF](https://www.ietf.org/)) started work on IPv6 in the early 1990s. The first RIR to run out of IPv4 addresses was ARIN (North America) in 2015, followed by the RIPE (Europe) in 2019, and LACNIC (South America) in 2020. The very last, free /8 address block of IPv4 addresses was issued by IANA in January 2011.
+
+The following realities contributed to the depletion of the IPv4 addresses:
+
+-   IPv4 addresses were designed to use 32 bits and are written with decimal numbers
+-   This allowed for 4.3 billion IP addresses
+
+The IPv4 address format is written in 4 groups of 4 numbers, each group separated by a period.
+
+Even though IPv4 addresses still trade hands, it’s actually quite difficult now to buy a completely unused block. What’s more, they’re expensive for smaller organizations (currently around $39 each) and leasing is cheaper. Unless you can acquire them from those sources, you’ll likely now be issued IPv6 ones.
```

Comment:
$39 doesn't seem like a lot of money.  Even an A-Block doesn't seem like a lot at about $10K. How do you make the price of a nice night out seem _intolerable_ for owning an IP address? (And do you own them? "Leasing is cheaper," sure, but how do you _own_ an IP address?)